libbeat/metric/system/cpu/cpu.go

```diff
@@ -116,7 +116,8 @@ func cpuPercentages(s0, s1 *sigar.Cpu, numCPU int) Percentages {
 	}
 
 	calculateTotalPct := func() float64 {
-		return common.Round(float64(numCPU)-calculatePct(s0.Idle, s1.Idle), common.DefaultDecimalPlacesCount)
+		idle := calculatePct(s0.Idle, s1.Idle) + calculatePct(s0.Wait, s1.Wait)
```

Comment:
Could you add a note to this line to explain why we calculate it like this? Can be later.

Reply:
Left a comment.